src/otx/engine/utils/auto_configurator.py

```diff
@@ -347,6 +347,7 @@ def update_ov_subset_pipeline(self, datamodule: OTXDataModule, subset: str = "te
         data_configuration = datamodule.config
         ov_test_config = self._load_default_config(model_name="openvino_model")["data"]["config"][f"{subset}_subset"]
         subset_config = getattr(data_configuration, f"{subset}_subset")
+        subset_config.batch_size = ov_test_config["batch_size"]
```

Comment:
Does `batch_size` also need to be updated here? and if so, could you include the batch size update in the warning message at the bottom of that code?

Reply:
Done!